packages/flutter_tts/tizen/src/text_to_speech.cc

```diff
@@ -131,6 +131,54 @@ std::vector<std::string> &TextToSpeech::GetSupportedLanaguages() {
   return supported_lanaguages_;
 }
 
+std::optional<std::pair<std::string, std::string>>
+TextToSpeech::GetDefaultVoice() {
+  char *language = nullptr;
+  int voice_type = 0;
+  int ret = tts_get_default_voice(tts_, &language, &voice_type);
+
+  if (ret != TTS_ERROR_NONE) {
+    LOG_ERROR("tts_get_default_voice failed: %s", get_error_message(ret));
+    if (language) free(language);
```

Comment:
nit: We usually assume the output values are only valid if the API call is successful (it's not our responsibility to release any value on function failure).

```suggestion
```